test/test-importer.js

```diff
@@ -29,6 +29,17 @@ module.exports = function (repo) {
       done()
     })
 
+    it('bad input', (done) => {
```

Comment:
Did you see 'small file as buffer (smaller than a chunk)'? It tests a Buffer that's <= 1 chunk.